src/components/Communicator/CommunicatorDialog/CommunicatorDialog.css

```diff
@@ -239,3 +239,7 @@
 .CommunicatorDialog__imageDialog__content img {
   max-width: 300px;
 }
+
+.selectable-text {
```

Comment:
We follow B.E.M. class naming convention (https://css-tricks.com/bem-101/), which is an acronym for Block, Element, Modifier.
Rename class name to: 
```
.CommunicatorDialog__boardInfoDialog
```